### PR TITLE
remove freactal dependency for vs modal + some clean up

### PR DESCRIPTION
**Pull request by evans-g-crsj**
image (6)

image

image

### CODE REVIEW

File: src/components/CohortBuilder/VirtualStudiesMenu/LoadQuery.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        <Tooltip
+          position="bottom"
+          open={this.state.open}
+          onRequestClose={() => {
+            this.setState({ open: false });
+          }}
+          interactive
+          html={
+            <div style={{ width: '200px' }}>
+              <React.Fragment>
```
Comment (francisl) [inline src/components/CohortBuilder/VirtualStudiesMenu/LoadQuery.js:50]: `<>`?

File: src/components/CohortBuilder/VirtualStudiesMenu/LoadQuery.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                {studies.map(({ virtualStudyId, name }) => {
+                  if (!selection || selection.virtualStudyId !== virtualStudyId) {
```
Comment (francisl) [inline src/components/CohortBuilder/VirtualStudiesMenu/LoadQuery.js:52]: ternary operator to remove some keywork here?

**Review by francisl — CHANGES_REQUESTED** [review]
Good Job, small comments.

File: src/components/CohortBuilder/VirtualStudiesMenu/LoadQuery.js (hunk 2 above)
Comment (evans-g-crsj) [reply src/components/CohortBuilder/VirtualStudiesMenu/LoadQuery.js:52]: Didn't pay attention to that but good point I'll clean taht up

**Review by francisl — APPROVED** [review]
:shipit:

**Pull request merged**